jovian/utils/latest.py

```diff
@@ -0,0 +1,36 @@
+import requests
+from random import random
+
+from jovian._version import __version__
+from jovian.utils.logger import log
+
+
+def get_latest_version():
+    """Returns Jovian's latest pypi version.
+    When Internet connection is not available it assumes 
+    the current version as the latest version.
+    """
+
+    pypi_json = 'https://pypi.org/pypi/jovian/json'
+    try:
+        latest_version = requests.get(pypi_json, timeout=3).json()['info']['version']
+    except:
+        latest_version = __version__
+
+    return latest_version
+
+
+def notify():
+    """Log Update available with current, latest version and command to upgrade.
+    """
+    latest_version = get_latest_version()
+    if latest_version > __version__:
```

Comment:
this won't work correctly. 
```
>>> "2.3.1" > "10.1.1"
True
```
instead use `packaging.version` to compare. see: https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

Reply:
This is requires install of `packaging` as its not default package.  Shall i continue with this or go a for a workaround ?